libs/wingc/src/jsify.rs

```diff
@@ -758,9 +822,127 @@ impl<'a> JSifier<'a> {
 				// This is a no-op in JS
 				CodeMaker::default()
 			}
-			StmtKind::Struct { .. } => {
-				// This is a no-op in JS
-				CodeMaker::default()
+			StmtKind::Struct { name, fields, extends } => {
+				// To allow for struct validation at runtime this will generate a JS class that has a static
+				// getValidator method that will create a json schema validator.
+
+				let mut code = CodeMaker::default();
+				let json_schema_require = "require(\"jsonschema\")".to_string();
```

Comment:
I'm wondering how this work in inflight, who makes sure that `jsonschema` will be bundled? I see there's a test for this, but don't understand how it works.

Reply:
Yea that was actually one of the challenges here since we don’t have other examples of bringing in external libraries for the inflight bundle, but if you checkout bundling.ts I added the sdk node modules as a resolvable path so esbuild can bundle it. Then esbuild does it’s magic when it sees the require() in the _toInflightType.

Reply:
Nice solution! I was struggling with something similar a while back and just gave up... So I guess we can now share external deps required by the language between inflight an preflight :clap:

Reply:
Yea Mark saved me from my bundling pit of sorrow 😁 Id like to get it to a whitelist sort of sharing, but that seems to be harder than it sounds. For now if its in the bundled dependencies of the SDK it can be used.